packages/calypso-e2e/src/secrets/secrets-manager.ts

```diff
@@ -220,6 +228,27 @@ export class SecretsManager {
 				appleLoginUser: {
 					...fakeAccount,
 				},
+				jetpackAtomicDefaultUser: {
+					...fakeFullAccount,
+				},
+				jetpackAtomicPhpOldUser: {
+					...fakeFullAccount,
+				},
+				jetpackAtomicPhpNewUser: {
+					...fakeFullAccount,
+				},
+				jetpackAtomicEcommPlanUser: {
+					...fakeFullAccount,
+				},
+				jetpackAtomicPrivateUser: {
+					...fakeFullAccount,
+				},
+				jetpackAtomicWpBetaUser: {
+					...fakeFullAccount,
+				},
+				jetpackAtomicWpPreviousUser: {
+					...fakeFullAccount,
+				},
```

Comment:
**thought:non-blocking** - I'll preface with "I know just enough TypeScript to make wil suggestions".

Is there a way to genericize this check? Having to make changes in three different places is error-prone and tedious.

Reply:
> Is there a way to genericize this check? Having to make changes in three different places is error-prone and tedious.

Heh, I'm totally there with you! 😆 

Yeah, there are ways, but they're actually not as easy or slick as one might hope... The problem is how types are handled in TypeScript -- they're entirely transpile-time and not runtime. As a result, it's kind of infamously annoying to try to do complicated runtime type validation (which is what this is) in TypeScript.

There are some libraries that help do that. Or you can also take care of this by writing recursive functions that crawl through every property and check it against the basic JavaScript types. Or, since this is JSON, we could actually maintain a JSON schema file and then just do a validation against that schema! In all these options, it's kind of a tradeoff of verbosity vs complexity. 

All that to say... I'm definitely interested in revisiting this, but I think we should push that out of the scope of this PR! 😁

Reply:
The main thing I could think of is turning it into a function which returns `fakeFullAccount` when you want to get one of these known users. Or generate the object with array.reduce over the known account slugs or something like that

(not a big deal obviously!)

Reply:
> Or generate the object with array.reduce over the known account slugs or something like that

Haha, yeah, this would be a relatively easy win and read a lot nicer, but right now the shapes of many of our users are all a little different. 🤦 I need to do a another clean-up or standardization of our secrets file again sometime soon...